src/test/scala/rise/elevate/movement.scala

```diff
@@ -21,7 +21,7 @@ class movement extends test_util.Tests {
   def betaEtaEquals(a: Rise, b: Rise): Boolean = {
     val na = BENF(a).get
     val nb = BENF(b).get
-    val uab: Rise = toBeTyped(na) :: nb.t
+    val uab: Rise = toBeTyped(na) !: nb.t
```

Comment:
Why use `!:` and not `::` here? It is not clear to me.

Reply:
Here they both behave the same. I have changed it back to `::`.